app/models/manageiq/providers/cloud_manager/provision_workflow/dialog_field_validation.rb

```diff
@@ -0,0 +1,9 @@
+# These methods are available for dialog field validation, do not erase.
+module ManageIQ::Providers::CloudManager::Provision::DialogFieldValidation
+  def validate_cloud_subnet(field, values, dlg, fld, value)
```

Comment:
@d-m-u looks like you missed the deletion of the original here: https://github.com/ManageIQ/manageiq/pull/8458/files#diff-affc070fa31fb03b3d1afa85434e8f3dL83